vavr/src-gen/main/java/io/vavr/Tuple1.java

```diff
@@ -152,7 +151,6 @@ public T1 _1() {
      * j=1
      */
     public <T2> Tuple2<T1, T2> concat(Tuple1<T2> tuple) {
-        Objects.requireNonNull(tuple, "tuple is null");
         return Tuple.of(_1, tuple._1);
```

Comment:
But we must check `tuple` for `null` because we will go deeper with them

Reply:
Oh you're right. Good call. I'll put just those back.